status/versions.go

```diff
@@ -73,8 +74,17 @@ func istioVersion() (*ExternalServiceInfo, error) {
 			version := expVersion.FindStringSubmatch(rawVersion)
 			if version != nil {
 				product.Version = version[0]
-				if !validateVersion(config.IstioVersionSupported, product.Version) {
-					info.WarningMessages = append(info.WarningMessages, "Istio version "+product.Version+" is not supported, the version should be "+config.IstioVersionSupported)
+				vSupported := config.Get().ExternalServices.Istio.IstioVersionSupported
+				isSupported := false
+				for i := 0; i < len(vSupported); i++ {
+					isSupported = validateVersion(vSupported[i], product.Version) || isSupported // 'OR' current check with previous to reset flag
```

Comment:
Without denoting what product this version belongs to, this version check is not going to be very good.

How does >1.0 and >0.1.0 work? If I'm running Istio 0.8.0 - this is going to pass, but it should not.

Or am I missing something additional this version check does?